### ISSUE TITLE
model.add_loss no longer works for tf.image.ssim_multiscale in tf 2.1

### ISSUE THREAD

**Issue by isaacgerg** [tensorflow/tensorflow#32564]
This tf.keras code was valid in tf 1.13.1:

`model.add_loss(tf.image.ssim_multiscale(reconstruction.inputs[0], reconstruction.outputs[0], 1.0))`
with types of reconstruction.inputs[0], reconstruction.outputs[0] being:
```
(<tf.Tensor 'input_1:0' shape=(?, 256, 256, 1) dtype=float32>,
 <tf.Tensor 'reconstruction_/truediv:0' shape=(?, 256, 256, 1) dtype=float32>)
```

but is no longer valid in tf 1.14.0 or 1.15.0rc0:

```
tensorflow.python.framework.errors_impl.InvalidArgumentError: 2 root error(s) found.
(0) Invalid argument: You must feed a value for placeholder tensor 'input_1' with dtype float and shape [?,256,256,1]
[[{{node input_1}}]]
[[MS-SSIM_1/Scale1/cond/Pad/paddings/_3891]]
(1) Invalid argument: You must feed a value for placeholder tensor 'input_1' with dtype float and shape [?,256,256,1]
[[{{node input_1}}]]
0 successful operations.
0 derived errors ignored.
```

I can not find any documentation on this change or how to fix it in the newer tf versions.

**Comment by ravikyram** [comment]
In order to expedite the trouble-shooting process, please provide a minimal standalone code to reproduce the issue reported here. Thanks!

**Comment by isaacgerg** [comment]
@ravikyram https://colab.research.google.com/gist/isaacgerg/5963501ac71cd7459a8b9e986386373d/tf_ssim_bug.ipynb

Toggle the pip install between tf version 1.13.1 and 1.14.0.

**Comment by ravikyram** [comment]
I have tried on colab with TF version 1.14.0 , 1.15.0rc0 and was able to reproduce the issue.However i am not seeing any issue with TF version 1.13.1.Thanks!

**Comment by isaacgerg** [comment]
Any update on this?

**Comment by isaacgerg** [comment]
Any update? Does this work in tf 2.0?

**Comment by isaacgerg** [comment]
@ravikyram @jvishnuvardhan Any update on this?

**Comment by isaacgerg** [comment]
This is still broken in tf 2.1

**Comment by Crispy13** [comment]
Any update?

**Comment by isaacgerg** [comment]
This problem is fixed as of tensorflow 2.4.  Thank you!
